### PR TITLE
Half-Duplex support

### LINKED ISSUE #519: Half-duplex mode support

### PR DESCRIPTION
**Pull request by dedmen**
Fixes #519
For review by other devs.

### CODE REVIEW

File: ts/src/plugin.cpp [hunk 1]
```diff
@@ -381,6 +381,21 @@ LISTED_INFO isOverLocalRadio(uint64 serverConnectionHandlerID, std::shared_ptr<C
 	result.waveZ = 1.0f;
 	if (senderData == NULL || myData == NULL) return result;
 
+	if (
+		!task_force_radio::config.get<bool>(Setting::full_duplex) &&
+		senderData->frequency.compare(serverIdToData[serverConnectionHandlerID].currentTransmittingFrequency)) {
+		/*
+		We are on half-duplex and currently transmitting on the incoming frequency. So we can't hear the other sender.
```
Comment (michail-nikolaev) [inline ts/src/plugin.cpp]: maybe pass radio id with freqs and use it for this problem?

**Comment by dedmen** [comment]
Done. But I'm tired and thats untested.
Also doesn't support DD radio because that doesn't actually have frequencies but instead only one string variable. But thats supposed to be removed in 1.0 anyway.

**Comment by treendy** [comment]
really like to use this feature, but been looking everywhere... no documentation or details on where this setting is set? (cant see it in the GUI, cant see any settings file in the pbo?)
How do i enable this feature?

**Comment by treendy** [comment]
ignore that... found it on steam workshop, beta version.
and it is all in the GUI! : )
Thanks for this amazing update, cant wait to try it out!
https://steamcommunity.com/sharedfiles/filedetails/?id=894678801

**Pull request merged**